### ISSUE TITLE
Input normalization

### ISSUE THREAD

**Issue by MathiasGilson** [ai-tor/Keras-ICNet#5]
why do you normalize the input with this function ?
  
https://github.com/aitorzip/Keras-ICNet/blob/bcc13bf5dd39abd3496dd669b245b91c31d7ff16/model.py#L16

It looks like an input value of 255 (let’s say in the green channel) would be normalized to 0.5 instead of 1 in that same channel.
I don’t understand why, can you help me ?

**Comment by pgr2015** [comment]
@MathiasGilson I guess the reason is because he want to rescale to range from 0-255 to -0.5-+0.5. The intuition behind this is similar to batchnormalization, which helps training to converge faster due to normalized linear projection.
